src/main/java/com/epam/aidial/core/util/RequestUtil.java

```diff
@@ -0,0 +1,82 @@
+package com.epam.aidial.core.util;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import lombok.experimental.UtilityClass;
+
+
+@UtilityClass
+public class RequestUtil {
```

Comment:
Please a javadoc explaining the purpose of the class

Reply:
Sure, I will